src/AspNet.Security.OAuth.Instagram/InstagramAuthenticationDefaults.cs

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using Microsoft.AspNet.Authentication;
+using Microsoft.AspNet.Authentication.OAuth;
+
+namespace AspNet.Security.OAuth.Instagram {
+    /// <summary>
+    /// Default values used by the Instagram authentication middleware.
+    /// </summary>
+    public static class InstagramAuthenticationDefaults {
+        /// <summary>
+        /// Default value for <see cref="AuthenticationOptions.AuthenticationScheme"/>.
+        /// </summary>
+        public const string AuthenticationScheme = "Instagram";
+
+        /// <summary>
+        /// Default value for <see cref="OAuthOptions.DisplayName"/>.
+        /// </summary>
+        public const string DisplayName = "Instagram";
+
+        /// <summary>
+        /// Default value for <see cref="OAuthOptions.ClaimsIssuer"/>.
+        /// </summary>
+        public const string Issuer = "Instagram";
+
+        /// <summary>
+        /// Default value for <see cref="OAuthOptions.CallbackPath"/>.
+        /// </summary>
+        public const string CallbackPath = "/signin-instagram";
+
+        /// <summary>
+        /// Default value for <see cref="OAuthOptions.AuthorizationEndpoint"/>.
+        /// </summary>
+        public const string AuthorizationEndpoint = "https://api.instagram.com/oauth/authorize";
+
+        /// <summary>
+        /// Default value for <see cref="OAuthOptions.TokenEndpoint"/>.
+        /// </summary>
+        public const string TokenEndpoint = "https://api.instagram.com/oauth/access_token";
+
+        /// <summary>
+        /// Default value for <see cref="OAuthOptions.UserInformationEndpoint"/>.
+        /// </summary>
+        public const string UserInformationEndpoint = "/users/self";
+
+        /// <summary>
+        /// Default location of the api
+        /// </summary>
+        public const string ApiLocation = "https://api.instagram.com/v1";
```

Comment:
This const won't be necessary when `UserInformationEndpoint` is updated to use an absolute URL :clap: